assets/javascripts/fullscreen.js

```diff
@@ -0,0 +1,27 @@
+function hideNavbar(fullscreen) {
```

Comment:
Since i'm using this function related to fullscreen on more than one place, i thought it would be better to make a seperate file for it